### DISCUSSION TITLE
Wiradjuri constellation art

### DISCUSSION THREAD

**Discussion by domcl** [Stellarium/stellarium#1833] [Q&A]
Hi! Just wondering if anyone is able to access the Wiradjuri constellation art? I'm trying to figure out where to find it, and have been able to find other constellation art/skylore, but not this particular package (but can find images from it so pretty sure it exists! 

https://www.industry.gov.au/news/our-indigenous-astronomical-traditions

Thanks in advance if anyone is able to help.

Cheers,
Dom

**Comment by gzotti** [comment]
This skyculture has not been contributed to the Stellarium project, so we are not aware of it. That page provides a contact address, though.
